.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: "\U0001F41BBug report"
+about: Create a report to help us improve Optuna.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Bug
+
+Please write a clear and concise description of what the bug is.
+
+## Expected behavior
+
+Please write a clear and concise description of what you expected to happen.
+
+## Environment 
+
+- Optuna version:
+- Python version:
+- OS:
+- Machine Learning library to be optimized:
```

Comment:
A nit but I think this could be stated optional, thinking of e.g. the official quadratic fit example. Also another nit but this sounds like the library itself should be optimized. How about abstracting it a bit, something along
```suggestion
- (Optional) Other libraries and their versions:
```

Reply:
Sounds good. Thanks.